admittance_controller/src/admittance_controller_parameters.yaml

```diff
@@ -0,0 +1,146 @@
+admittance_controller:
+  joints: {
+    type: string_array,
+    description: "specifies which joints will be used by the controller",
+    read_only: true
+  }
+
+  command_interfaces:
+    {
+      type: string_array,
+      description: "specifies which command interfaces to claim",
+      read_only: true
+    }
+
+  state_interfaces:
+    {
+      type: string_array,
+      description: "specifies which state interfaces to claim",
```

Comment:
Please write the comments in sentence form and make them clearer, as I showed in the proposals above.